drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/editor/GuidedDecisionTableGraphEditorPresenter.java

```diff
@@ -336,6 +345,20 @@ public void init() {
                 removeDocument( dtPresenter );
             }
         } );
+
+        registeredDocumentsMenuBuilder.setNewDocumentCommand( this::onNewDocument );
+    }
+
+    void onNewDocument() {
+        final Path contextPath = context.getActivePackage().getPackageMainResourcesPath();
+        helper.createNewGuidedDecisionTable( contextPath,
+                                             "",
+                                             GuidedDecisionTable52.TableFormat.EXTENDED_ENTRY,
+                                             view,
+                                             ( path ) ->
+                                                     onOpenDocumentsInEditor( new ArrayList<Path>() {{
```

Comment:
Collections.singletonList(path) would be much better than the double brace initialization which creates an anonymous class every time.